examples/fluidsim.dx

```diff
@@ -1,6 +1,6 @@
 '## Fluid sim
 Fluid simulation code based on
-[Real-Time Fluid Dynamics for Games](https://www.josstam.com/publications) by Jos Stam
+[Float-Time Fluid Dynamics for Games](https://www.josstam.com/publications) by Jos Stam
```

Comment:
"float-time fluid dynamics": it's fluid dynamics, but with things floating in it